modules/enterprise/gui/portal-war/src/main/java/org/rhq/enterprise/gui/common/paging/PageControlSettingsUIBean.java

```diff
@@ -24,8 +24,25 @@
  * @author Joseph Marques
  */
 public class PageControlSettingsUIBean {
-    private SelectItem[] pageSizes = new SelectItem[] { new SelectItem("15", "15"), new SelectItem("30", "30"),
-        new SelectItem("45", "45") };
+    private SelectItem[] pageSizes;
+    private static String MAX_ITEMS_PER_PAGE = "rhq.server.gui.max-items-per-page";
+
+    public PageControlSettingsUIBean() {
+        try {
+            String maxItemsPerPage = System.getProperty(MAX_ITEMS_PER_PAGE);
+            if (maxItemsPerPage != null && Integer.parseInt(maxItemsPerPage) > 45) {
+                pageSizes = new SelectItem[] { new SelectItem("15", "15"), new SelectItem("30", "30"),
```

Comment:
Do we still have those UI pages where the scrolling starts to be buggy if there's more than X items per page? I'm not sure if all of those were fixed..